gateleen-hook/README_hook.md

```diff
@@ -248,7 +248,7 @@ Gateleen allows searching for listeners and routes using the query parameter `q`
 Search for listeners based on a query parameter like this:
 
 ```
-GET http://myserver:7012/gateleen/server/listenertest/_hooks/listeners/listener/1?q=testQuery
+GET http://myserver:7012/gateleen/server/listenertest/_hooks/listeners?q=testQuery
```

Comment:
Running locally it was possible to list using
http://localhost:7012/listeners?q=test
or in the case of routes
http://localhost:7012/routes?q=test
This is because the request arrives at handle, the currently validation checks only if contains the LISTENERS_KEY or the ROUTES_KEY (is that correct?)

Anyway, I updated the suggested url.

Reply:
> Running locally it was possible to list using http://localhost:7012/listeners?q=test or in the case of routes http://localhost:7012/routes?q=test This is because the request arrives at handle, the currently validation checks only if contains the LISTENERS_KEY or the ROUTES_KEY (is that correct?)
> 
> Anyway, I updated the suggested url.

No this is not correct. Please read my remarks. I already commented that it's wrong to rely on "contains 'listeners'" only.